README.md

```diff
@@ -1,18 +1,176 @@
+contributors.
 
-### Adding a new crate to rust-vmm
+Each virtualization component lives in a separated GitHub repository under the
+rust-vmm organization. The code is organized such that one repository
+corresponds to one
+[Rust crate](https://doc.rust-lang.org/book/ch07-01-packages-and-crates-for-making-libraries-and-executables.html).
 
-Each rust-vmm crate lives in its own repo under the [rust-vmm organization](http://github.com/rust-vmm).
-If you want to contribute to rust-vmm by adding a new Rust crate to the project, you will have to go through the following steps:
+## Why rust-vmm?
 
-1. Submit a [crate addition request](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=new-crate-request.md&title=Crate+Addition).
-2. The crate relevance to the project will be discussed through the issue submitted in step 1.
-3. If the crate gets accepted by the project maintainers, a new, empty repository will be created under the rust-vmm repository.
-4. Submit a pull request to the newly created repository as your initial code submission.
+- **Reduce code duplication**. The initial thought with rust-vmm was to create
+  a place for sharing common virtualization components between two existing
+  VMMs written in Rust:
+  [CrosVM](https://chromium.googlesource.com/chromiumos/platform/crosvm/) and
+  [Firecracker](https://github.com/firecracker-microvm/firecracker/). These
+  two projects have similar code for calling KVM ioctls, managing the
+  Virtual Machine memory, interacting with virtio devices and others. Instead
+  of having these components live in each of the projects repositories, they
+  can be shared through the rust-vmm project.
+- **Faster development**. rust-vmm provides a base of virtualization components
+  which are meant to be generic such that they can be consumed by other
+  projects besides CrosVM and Firecracker. One example that is often mentioned
+  is building a container specific VMM. By doing so, the container VMM can
+  reuse most of the rust-vmm components and simply build the glue around them
+  as well as an appropriate API for interacting with the container.
+- **Security & Testability**. Having independent components makes fuzz testing
+  easy to apply to each individual package. Our top priority is now
+  [vm-virtio](https://github.com/rust-vmm/vm-virtio) as it provides a
+  [virtio](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=virtio)
+  device implementation. We want to keep a high standard in terms of testing
+  as these virtualization packages are going to be used in production by
+  multiple projects. Each component is individually tested with a set of
+  common integration tests responsible for running unit tests and linters
+  (including coding style checks), and computing the coverage. In the future
+  we are planning to test the integration of the rust-vmm components by
+  providing a reference VMM implementation.
```

Comment:
What do you mean? It will be used only for testing the integration of components. People will still need to have their own tests in their VMM. Should I mention that here?

Reply:
I think so, it's good to clearly say what rust-vmm will test and what it will not test.